__tests__/games.test.js

```diff
@@ -285,4 +285,66 @@ afterAll(() => {
     
   });
 
-  
\ No newline at end of file
+describe('GET /api/reviews/:review_id/comments', () => {
+  test('200: responds with an array of comments for the given review_id', () => {
+    return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then((results) => {
+
+          const comments = results.body.comments
+
+          expect(comments.length).toBe(3);
+          comments.forEach((comment) => {
+              expect(comment).toMatchObject({
+                review_id: expect.any(Number),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                body: expect.any(String),
+                author: expect.any(String),
+                comment_id: expect.any(Number)
+
+              })
+
+          })
+
+        })
+  });
+  test('200: responds with an empty array if the review exists but there is no comments to show', () => {
+    return request(app)
+    .get("/api/reviews/1/comments")
+    .expect(200)
+    .then((results) => {
+
+      const comments = results.body.comments
```

Comment:
I'd recommend you're consistent with the "style" of destructuring - in other tests you destructure inline eg `.then(({body}) => {` either is fine, but aim to be consistent.